src/rules/selectorsFormatRule.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (c) 2019-present, ProductBoard, Inc.
+ * All rights reserved.
+ */
+
+import * as Lint from 'tslint';
+import * as ts from 'typescript';
+
+export class Rule extends Lint.Rules.AbstractRule {
+  public apply(sourceFile: ts.SourceFile): Array<Lint.RuleFailure> {
+    return this.applyWithWalker(
+      new SelectorsFormatRule(sourceFile, this.getOptions()),
+    );
+  }
+}
+
+type Configuration = {
+  reference: string;
+  // TODO: do we wanna make it optional? Does fallback on line `68` make sense?
+  importPaths?: { [key: string]: string | Array<string> };
```

Comment:
Since even in our project we won't use it undefined, it would make it mandatory. To simplify things, what do you think? 🙏